packages/cli/src/commands/CreateDatabaseCommand.ts

```diff
@@ -0,0 +1,24 @@
+import { Arguments, CommandModule } from 'yargs';
+import { MikroORM } from '@mikro-orm/core';
+import { AbstractSqlDriver, SchemaGenerator } from '@mikro-orm/knex';
+import { CLIHelper } from '../CLIHelper';
+
+export class CreateDatabaseCommand implements CommandModule {
+
+  command = 'database:create';
+  describe = 'Create your database if it does not exist';
+
+  /**
+   * @inheritdoc
+   */
+  async handler(args: Arguments) {
+    const orm = await CLIHelper.getORM() as MikroORM<AbstractSqlDriver>;
+    const { em } = orm;
+
+    const schemaGenerator = new SchemaGenerator(em);
```

Comment:
just a small nitpick, this can be shortened

```suggestion
    const schemaGenerator = orm.getSchemaGenerator();
```